src/main/java/org/folio/rest/impl/InventoryHelper.java

```diff
@@ -385,14 +336,14 @@ private String buildLookupEndpoint(String type, Object... params) {
    * @param productTypesMap product types Map used to build Inventory query
```

Comment:
Please also remove `productTypesMap` from methods Javadocs where it was available before

Reply:
Agree, removed.